test/taoensso/faraday/tests/requests.clj

```diff
@@ -219,6 +233,13 @@
     (expect "global-secondary" (.getIndexName action))
     ))
 
+(let [req ^UpdateTableRequest (update-table-request
```

Comment:
Interesting, I stand corrected. This may be a compiler change that happened sometime without me noticing, or I may just be misremembering; was under the incorrect impression that type metadata wasn't in general inferred from a form to a local binding. Thanks for pointing this out!